### DISCUSSION TITLE
Form Request Validation

### DISCUSSION THREAD

**Discussion by mhafizkn** [spatie/laravel-settings#108] [Q&A] [answered]
Hi, I'm just exploring and using this package. I want to validate the data inserted by the user using function rules().

Therefore, it is possible to use Validation: Form Request Validation?

Thank you.

**Comment by rubenvanassche** [comment, marked as answer]
At the moment we do not have such functionality, but I would merge a PR where a `rules` function on the settings object would first validate all the settings before saving the settings and eventually throw out validation errors.

**Reply by mhafizkn** [reply]
Thanks for the reply!
